aten/src/ATen/native/Convolution.cpp

```diff
@@ -157,7 +157,7 @@ static void check_input_shape_forward(const at::Tensor& input,
 
   AT_CHECK(weight_dim == k,
            "Expected ", weight_dim, "-dimensional input for ", weight_dim,
-           "-dimensional weight ", weight.sizes(), ", but got input of size ",
+           "-dimensional weight ", weight.sizes(), ", but got ", weight_dim, "-dimensional input of size ",
```

Comment:
Is weight_dim the thing we want here? Or something in the lines of input.sizes().size()?